client/src/components/MenueCard.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import PropTypes from 'prop-types';
+
+const InputContainer = styled.div`
```

Comment:
why is it called an `InputContainer` I don't see any `<input>` elements 🧐

Reply:
Your right it isn't very obvious. I changed that :)